data/aws_resources_specification.json

```diff
@@ -25262,7 +25262,8 @@
         "InstanceAttributes": {
```

Comment:
Interesting... As a new spec version was available I downloaded 2.17.0, and reran your test and it worked. Though that doesn't quite make sense in relation to the test you've added and the final diff to master... I will have a look at this tomorrow and definitely release a version which supports tagging for SQS Queues! Sorry about the delay!